reference-implementation/lib/parser.js

```diff
@@ -122,3 +128,17 @@ function normalizeAddress(address, baseURL) {
 function isJSONObject(value) {
   return typeof value === 'object' && value !== null && !Array.isArray(value);
 }
+
+function longerLengthThenCodeUnitOrder(a, b) {
+  return compare(b.length, a.length) || compare(a, b);
+}
+
+function compare(a, b) {
+  if (a > b) {
+    return 1;
+  }
+  if (b > 1) {
```

Comment:
Does this mean we don't sort `a` and `b` alphabetically in the import map, but we do sort `aa` and `bb` alphabetically?

Reply:
This is a typo! Should be `b > a`. Now to write a test that exposes this...

Reply:
👍